phonon/update.py

```diff
@@ -68,8 +70,6 @@ def __init__(self, process, _id, database='test', collection='test',
         self.soft_session = soft_session
         self.soft_expiration = datetime.datetime.now(LOCAL_TZ) + datetime.timedelta(seconds=self.soft_session)
         self.hard_expiration = datetime.datetime.now(LOCAL_TZ) + datetime.timedelta(seconds=self.hard_session)
-        if self.init_cache:
-            self.__cache()
```

Comment:
why'd you take the `init_cache` logic out of the `BaseUpdate` class? Looks like it's implemented exactly the same in the in both the sub-classes.